src/styles/index.scss

```diff
@@ -15,18 +13,71 @@ code {
 }
 
 body, html, .lg-container {
-  height: 100%;
   margin: 0;
   right: 0;
   top: 0;
+}
+
+.lg-container {
+  display: flex;
+  flex-direction: column;
   overflow-x: hidden;
-  text-align: center;
   list-style: none;
 }
 
-.header {
+
+.container {
   width: 100%;
   height: 100vh;
   background-color: $mainColor;
+  color: $secondaryColor;
   padding: 50px;
+
+  button {
+    padding: 0;
+    border: none;
+    background: none;
+    background-color: $lightMainColor;
+    color: $secondaryColor;
+    font-size:15px;
+    padding: 10px 15px;
+  }
+
+}
+
+.title {
+  margin-left: 20%;
+  text-transform: uppercase;
+  
+
+  .firstTitle{
+    font-style: italic;
+    font-size: 55px;
+    font-weight: 200;
+    margin-left: -170px;
+    margin-bottom: -40px;
+  }
+
+  .secondTitle{
+    font-size: 70px;
+    font-weight: 900;
+    top: -100px;
+  }
+
+}
+
+.text{
+  margin-top: -36px;
+  margin-left: 20%;
+  color: $lightPink;
+  line-height: 1.6;
+  font-weight: 200px;
+  width: 400px;
+}
+
+// ----- ONTWERP TECHNIEKEN ----- 
+
+.designTech{
```

Comment:
Deze className kan generieker, daarnaast zie ik hem nergens meer in de code. Werkt deze dus, of is hij overbodig?

Reply:
Ja hier had ik nog niks mee gedaan, kan inderdaad verwijderd worden. Goed opgemerkt!